catalog-info.yaml

```
@@ -13,6 +13,7 @@ metadata:
       title: Grimoire Pull Request Backlog
       icon: dashboard
+  subcomponentOf: LTI
 spec:
   type: XBlock
   lifecycle: production
```

```suggestion

spec:
  type: XBlock
  lifecycle: production
  subcomponentOf: LTI
```

The `subcomponentOf` attribute is a part of the `spec`